frontend/src/containers/ForgotPasswordPage/ForgotPasswordPage.js

```diff
@@ -1,8 +1,65 @@
 import React from 'react';
+import { Grid } from '@material-ui/core';
+import ForgotPasswordCard from '../../components/ForgotPasswordCard/ForgotPasswordCard';
+import CustomSnackbar from '../../components/CustomSnackbar/CustomSnackbar';
+import { sendForgotPasswordEmail } from '../../api/PublicAPI';
 
 class ForgotPasswordPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showServerError: false,
+      showSuccessMessage: false,
+    };
+  }
+
+  handleSendClick = async (email) => {
+    try {
+      const response = await sendForgotPasswordEmail(email);
+      if (response.status !== 200) {
```

Comment:
We are already using ".then" and ".catch". Maybe we should stick with those or agree on a standard?